### DISCUSSION TITLE
How to replace set_value in Keras 3?

### DISCUSSION THREAD

**Discussion by santi-mir** [keras-team/keras#19824] [Q&A] [answered]
I can see that it's deprecated (also in the repo code)

How do we replace `set_value` ?
https://www.tensorflow.org/api_docs/python/tf/keras/backend/set_value

**Comment by abhaskumarsinha** [comment, marked as answer]
Hey there,

You can still use *TensorFlow*'s functions to set values within the Keras v3.0+ framework. 

To work out in backend invarient way, you might convert the weights to numpy tensors and then set them as weights back depending upon the requirement and size of the model.

Thank You.

Best Regards.

**Reply by santi-mir** [reply]
Thanks, yes I'm doing `Variable.assign()`
